generation/scraper/Partitions/DirectXShaderCompiler/settings.rsp

```diff
@@ -0,0 +1,11 @@
+--file
+<PartitionDir>\main.cpp
+--output
+<GeneratedSourceDir>\<PartitionName>.cs
+--include-directory
+<RepoRoot>\generation\scraper
+<IncludeRoot>/um
+--traverse
+<IncludeRoot>/um/dxcapi.h
+--namespace
+Microsoft.DirectXShaderCompiler
```

Comment:
Is this the right namespace? I'm not sure :)

Reply:
Please move this to Windows.Win32.Graphics.Hlsl.

Reply:
@mikebattista Updated to that, and regenerated the baseline too.